core/src/main/resources/error/error-classes.json

```diff
@@ -54,13 +90,25 @@
     "message" : [ "The fraction of sec must be zero. Valid range is [0, 60]." ],
     "sqlState" : "22023"
   },
+  "INVALID_JDBC_NUMPARTITIONS" : {
+    "message" : [ "Invalid value `%s` for parameter `%s` in table writing via JDBC. The minimum value is 1." ],
+    "sqlState" : "42000"
+  },
+  "INVALID_JDBC_TXN_ISOLATION_LEVEL" : {
```

Comment:
As the error-class is user-facing, we should avoid using abbreviations. How about `TRANSACTION` instead of `TXN`?